versioned_docs/version-2.8/reference-guides/about-the-api/api-tokens.md

```diff
@@ -23,7 +23,6 @@ Here is the complete list of tokens that are generated with `ttl=0`:
 
 | Token             | Description                                                                            |
 | ----------------- | -------------------------------------------------------------------------------------- |
-| `kubeconfig-*`    | Kubeconfig token                                                                       |
```

Comment:
Did we change the name of the tokens created when `kubeconfig-generate-token` setting is `false`

Reply:
nvm, I didn't see the heading of this section this removal is correct.